logos/icon.svg

```diff
@@ -1,18 +1,18 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Why are the logos all changed? suspect it is line endings

Reply:
doesn't seem to have caused any material problems though. It views OK.